File: bongf/java/src/level1/DivisibleNumArray.java
package level1;

import java.util.Arrays;

public class DivisibleNumArray {
    public static void main(String[] args) {
        int[] arr = {3, 2, 6};
        int divisor = 10;
        System.out.println(Arrays.toString(solution(arr, divisor)));
    }

    public static int[] solution(int[] arr, int divisor) {
        int[] ints = Arrays.stream(arr).filter(x -> x % divisor == 0).sorted().toArray();
        if (ints.length == 0) {
            return new int[]{-1};
        }
        return ints;
    }
}

File: programmers/python/level1/divisible_num_array.py
def solution(arr, divisor):
    answer = []
    for n in arr :
        if n % divisor == 0 :
            answer.append(n)
    return [-1] if len(answer) == 0 else sorted(answer) 

# 파이썬에서 0, [], " ", False 등은 거짓 그 외에 나머지 값은 참
def solution2(arr, divisor):
    return sorted([n for n in arr if n % divisor == 0]) or [-1]

arr = [3,2,6]
divisor = 10
print(solution2(arr, divisor))